app/src/controllers/organizations/projects/selectors.js

```diff
@@ -25,7 +28,24 @@ export const projectsPaginationSelector = (state) => domainSelector(state).pagin
 export const projectsSelector = (state) => domainSelector(state).projects;
 export const loadingSelector = (state) => domainSelector(state).loading || false;
 
-export const querySelector = createQueryParametersSelector({
+export const createOrganizationProjectsParametersSelector = ({
+  defaultPagination,
+  defaultSorting,
+  sortingKey,
+} = {}) =>
+  createSelector(
+    createQueryParametersSelector({
+      defaultPagination,
+      defaultSorting,
+      sortingKey,
+    }),
+    ({ [SIZE_KEY]: limit, [SORTING_KEY]: sort, [PAGE_KEY]: pageNumber }) => {
+      return getAlternativePaginationAndSortParams(sort, limit, pageNumber);
+    },
+  );
+
+export const querySelector = createOrganizationProjectsParametersSelector({
+  defaultPagination: DEFAULT_PAGINATION,
   defaultFields: [DEFAULT_SORT_COLUMN],
```

Comment:
How these fields are used within the query selector?

Reply:
Removed